generation/core/generator.js

```diff
@@ -81,13 +89,22 @@ module.exports = function({
 	}
 
 	function createMethod(classJson, json) {
+		// For us this means that we don't set the class type, but let it be set from the outside
+		// This is an assumption based on our current experience with EarlGrey, we might need to rework this at some point
+		const isTranslucent = !json.static;
```

Comment:
what does `translucent` mean in this context ?

Reply:
It means that we don

Reply:
Not sure I got it, can you elaborate?

Reply:
Sorry, pressed enter too early, does this make more sense?